scripts/config/msu/semver.nut

```diff
@@ -99,25 +99,32 @@
 		return ret;
 	}
 
+	function verifyCompareInputs( _version1, _version2 ) // private
```

Comment:
Consider making a local function instead of a private one as this function is not accessed outside this file.

Reply:
yeah local function is a good call